test/helpers_test.rb

```diff
@@ -0,0 +1,29 @@
+require 'test_helper'
+
+describe Lotus::Assets::Helpers do
+
+  def safe_string_class
+    ::Lotus::Utils::Escape::SafeString
+  end
+
+  describe '#javascript' do
+    before do
+      @javascript = View.new.javascript('feature-a')
+    end
+
+    it 'returns an instance of SafeString' do
+      @javascript.must_be_instance_of safe_string_class
```

Comment:
What do you think about checking that the string does not escaped here?

We care about that more than what kind of instance it is